resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -128,6 +130,30 @@ if [[ "$os" = 'el' || "$os" = 'sles' || "$os" = 'fedora' ]]; then
                 params+=("--additional-dependency")
                 params+=("<%= dep -%>")
         <% end -%>
+        # get rpm install trigger scripts
+        <% EZBake::Config[:redhat][:postinst_install_triggers].each do |package, script|
+             rpm_paths.each do |path| -%>
+              cat > <%=path %>/<%=package%>_postinst_triggers <<EOM
+              if [ "\$1" -eq 1 ] ; then
+                 "<%= script.join("\n") %>"
+                 :
+              fi
+EOM
+            <% end -%>
+            params+=("--rpm-trigger" "'<%=package%>: $dir/<%=package%>_postinst_triggers'")
+        <% end -%>
+        # get rpm upgrade trigger scripts
+        <% EZBake::Config[:redhat][:postinst_upgrade_triggers].each do |package, script|
+             rpm_paths.each do |path| -%>
+             cat > <%=path %>/<%=package%>_postinst_triggers <<EOM
+              if [ "\$1" -eq 1 ] ; then
```

Comment:
@mwaggett ah, good catch. @sarameisburger this should be `if [ "\$1" -gt 1 ]; then`

Reply:
ohhh yes the classic copypasta mistake

Reply:
Can you explain what these are checking for / why these checks are being used?

Reply:
These checks are to tell whether this is an installation or an upgrade. This website has a handy chart http://meinit.nl/rpm-spec-prepostpreunpostun-argument-values

Reply:
ahh, okay, that chart helps, thanks!